protocols/api/src/main/java/org/apache/james/protocols/api/handler/MultiLineHandler.java

```diff
@@ -1,67 +0,0 @@
-/****************************************************************
- * Licensed to the Apache Software Foundation (ASF) under one   *
- * or more contributor license agreements.  See the NOTICE file *
- * distributed with this work for additional information        *
- * regarding copyright ownership.  The ASF licenses this file   *
- * to you under the Apache License, Version 2.0 (the            *
- * "License"); you may not use this file except in compliance   *
- * with the License.  You may obtain a copy of the License at   *
- *                                                              *
- *   http://www.apache.org/licenses/LICENSE-2.0                 *
- *                                                              *
- * Unless required by applicable law or agreed to in writing,   *
- * software distributed under the License is distributed on an  *
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
- * KIND, either express or implied.  See the License for the    *
- * specific language governing permissions and limitations      *
- * under the License.                                           *
- ****************************************************************/
-package org.apache.james.protocols.api.handler;
-
-import java.nio.ByteBuffer;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-
-import org.apache.james.protocols.api.ProtocolSession;
-import org.apache.james.protocols.api.ProtocolSession.State;
-import org.apache.james.protocols.api.Response;
-
-/**
- * A special {@link LineHandler} which will "buffer" the received lines till a point and the push them all at
- * one to the {@link #onLines(ProtocolSession, Collection)} method
- * 
- *
- * @param <S>
- */
-public abstract class MultiLineHandler<S extends ProtocolSession> implements LineHandler<S> {
```

Comment:
strange, it is used in protocols/imap/src/main/java/org/apache/james/protocols/imap/core/IMAPCommandDispatcher.java , no?